Adds SPM support (#97)
iosphere/ISHPullUp: README.md

* `ISHPullUp/ISHPullUpRoundedView.`{`h`/`m`}
* `ISHPullUp/ISHPullUpViewController.`{`h`/`m`}

### Swift Package Manager

__NOTE__: _These instructions are intended for usage on Xcode 11 and higher. Xcode 11 is the first version of Xcode that integrates Swift Package manager and makes it way easier to use than it was at the command line. If you are using older versions of Xcode, we recommend using CocoaPods._

1. Go to File > Swift Packages > Add Package Dependency... (If you have multiple projects, select the project you want to add the library to)

> 2. Paste the URL to the LTHPasscodeViewController repo on GitHub (https://github.com/iosphere/ISHPullUp) into the search bar, then hit the Next button:

Reviewer: ```suggestion
2. Paste the URL to the [this repo](https://github.com/iosphere/ISHPullUp) into the search bar, then hit the Next button:
```

Author: Great catch, thanks! I've been doing a few of these and got a little too comfortable with the copy and pasting 😅
